src/Manifest/ManifestManager.php

```diff
@@ -61,11 +61,12 @@ public function getFileManifest(string $fileName): array
     /**
      * @return mixed
      */
-    public function getTableManifest(string $tableName)
+    public function getTableManifest(string $tableName, bool $asObject = false)
     {
         $baseDir = implode('/', [$this->dataDir, 'in', 'tables']);
+        $manifestArray = $this->loadManifest($tableName, $baseDir);
 
-        return $this->loadManifest($tableName, $baseDir);
+        return $asObject ? ManifestOptions::fromArray($manifestArray) : $manifestArray;
```

Comment:
K čemu to je vracet to jako pole? To bude akorát bordel

Reply:
Zpětná kompatibilita. Ale asi netřeba jí řešit no.